src/views/search/search.jsx

```diff
@@ -138,4 +144,12 @@ var Search = injectIntl(React.createClass({
     }
 }));
 
-render(<Page><Search /></Page>, document.getElementById('app'));
+var mapStateToProps = function (state) {
+    return {
+        navigationDetails: state.navigationDetails
```

Comment:
If the state is called `searchTerm` elsewhere, it should be called `searchTerm` here too